rustls-cert-gen/Cargo.toml

```diff
@@ -6,13 +6,19 @@ license.workspace = true
 edition.workspace = true
 keywords.workspace = true
 
+[features]
+default = ["ring"]
+aws_lc_rs = ["dep:aws-lc-rs", "rcgen/aws_lc_rs"]
+ring = ["dep:ring", "rcgen/ring"]
+
 [dependencies]
-rcgen = { path = "../rcgen", default-features = false, features = ["pem", "ring"] }
+rcgen = { path = "../rcgen", default-features = false, features = ["pem"] }
 bpaf = { version = "0.9.5", features = ["derive"] }
 pem = { workspace = true }
-ring = { workspace = true }
+ring = { workspace = true, optional = true }
 rand = { workspace = true }
 anyhow = "1.0.75"
+aws-lc-rs = { version = "1.6.0", optional = true }
```

Comment:
I think we should share the aws-lc-rs version throughout the workspace, like `ring`.